js/index.js

```diff
@@ -40,3 +40,104 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const test = siteContent.nav["nav-item-1"];
+console.log(test);
+
+const anchors = document.getElementsByTagName('a');
```

Comment:
Here we can use a more specific selector such as `nav a`. You never know when more links might get added to the page. 👍